lib/src/components/NxToggle/types.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { LabelHTMLAttributes } from 'react';
+import * as PropTypes from 'prop-types';
+
+export type Props = Omit<LabelHTMLAttributes<HTMLLabelElement>, 'onChange'> & {
+  toggleId?: string | null;
```

Comment:
I feel like this should still be `inputId` as the purpose of it is to kind of get under the covers and put an id on the `<input>` element, not some more abstract "toggle" element.

Reply:
> I feel like this should still be `inputId` as the purpose of it is to kind of get under the covers and put an id on the `<input>` element, not some more abstract "toggle" element.

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/c4d968435f7dcb9535d225bae8362ae80cb03b90